docs/installation.md

```diff
@@ -388,87 +335,87 @@ Get the logs from that pod:
 ```
 
-### Prometheus Troubleshooting
-
-Please refer to [Troubleshooting section in Prometheus document](/docs/prometheus.md#troubleshooting).
-
-## Upgrading Sumo Logic Collection
-
-To upgrade our helm chart to a newer version, you must first run update your local helm repo.
-
-```bash
-helm repo update
-```
-
-Next, you can run `helm upgrade --install` to upgrade to that version. The following upgrades the current version of `my-release` to the
-latest.
+## Installing in OpenShift platform
 
-```bash
-helm upgrade --install my-release sumologic/sumologic -f `user-values.yaml`
-```
+The daemonset/statefulset fails to create the pods in Openshift environment due to the request of elevated privileges, like HostPath mounts,
+privileged: true, etc.
 
-If you wish to upgrade to a specific version, you can use the `--version` flag.
+If you wish to install the chart in the Openshift Platform, it requires a SCC resource which is only created in Openshift (detected via API
+capabilities in the chart), you can add the following configuration to `user-values.yaml`:
 
-```bash
-helm upgrade --install my-release sumologic/sumologic -f `user-values.yaml` --version=2.0.0
+```yaml
+sumologic:
+  scc:
+    create: true
+otellogs:
+  daemonset:
+    containers:
+      otelcol:
+        securityContext:
+          privileged: true
+    initContainers:
+      changeowner:
+        securityContext:
+          privileged: true
+tailing-sidecar-operator:
+  scc:
+    create: true
 ```
 
-**Note:** If you no longer have your `user-values.yaml` from the first installation or do not remember the options you added via `--set` you
-can run the following to see the values for the currently installed helm chart. For example, if the release is called `my-release` you can
-run the following.
+**Notice:** Prometheus Operator is deployed by default on OpenShift platform, you may either limit scope for Prometheus Operator installed
+with Sumo Logic Kubernetes Collection using `kube-prometheus-stack.prometheusOperator.namespaces.additional` parameter in `user-values.yaml`
+or exclude namespaces for Prometheus Operator installed with Sumo Logic Kubernetes Collection using
+`kube-prometheus-stack.prometheusOperator.denyNamespaces` in `user-values.yaml`. For details see
+[Prometheus document](/docs/prometheus.md#prometheus-operator-in-the-cluster)
 
-```bash
-helm get values my-release
-```
+## Non-helm installation
 
-If something goes wrong, or you want to go back to the previous version, you can
-[rollback changes using helm](https://helm.sh/docs/helm/helm_rollback/):
+If you don't want to use Helm to manage the installation, please use `helm template` to generate Kubernetes templates and apply them using
+Kubectl.
 
-```
-helm history my-release
-helm rollback my-release <REVISION-NUMBER>
-```
+> **Warning:** > Before upgrade, please delete the old jobs:
 
-## Uninstalling Sumo Logic Collection
+- `kubectl delete job -n ${NAMESPACE} my-release-sumologic-setup`
```

Comment:
why do we need to delete the job explicitly, isn't the job/pod removed once its complete?

Reply:
It is after some TTL (https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/), but this isn't guaranteed to have passed at this point.